test/by-patron-type.test.js

```diff
@@ -32,4 +32,13 @@ describe('by-patron-type', function () {
       expect(patronType.accessibleDeliveryLocationTypes).to.not.be.empty
     }
   })
+  it('will have deliverableTo property for scholar p types', function () {
+    for (const pType in this.byPatronType) {
+      if (this.byPatronType[pType].scholarRoom) {
+        expect(this.byPatronType[pType].scholarRoom).to.not.be.empty
+        expect(this.byPatronType[pType].scholarRoom.code).to.be.a.string
```

Comment:
This isn't testing the type. I don't think this assert style is available here. I think you need to use full functions (e.g. `.to.be.a('string')`)